superbench/runner/playbooks/deploy.yaml

```diff
@@ -25,16 +25,26 @@
   tasks:
     - name: Checking NVIDIA GPU
```

Comment:
It's not check NVIDIA GPU, right? If you would like to check, you should use lspci.

Reply:
renamed